app/controllers/registers_controller.rb

```diff
@@ -31,21 +31,13 @@ def recover_records(fields, params)
 
     sort_by = params[:sort_by] ||= default_sort_by.call
     sort_direction = params[:sort_direction] ||= 'asc'
```

Comment:
would be possible to put these default assignments into the model as well if that is preferable.